packages/core/src/components/TextField/TextField.example.jsx

```diff
@@ -18,12 +18,7 @@ ReactDOM.render(
       multiline
       name="multiline_example"
     />
-    <TextField
```

Comment:
Why remove the disabled example?

Reply:
Ah, I originally replaced it with the currency example before I created a separate page for it. Just added the disabled example back.